scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/grid/jobs/JobsListGrid.java

```diff
@@ -102,17 +104,27 @@ protected void selectionChangedHandler(SelectionEvent event) {
         }
     }
 
+    @Override
+    protected void selectionUpdatedHandler(SelectionUpdatedEvent event) {
+        ListGridRecord[] selectedRecords = this.getSelectedRecords();
+        ArrayList<Integer> selectedJobsIds = new ArrayList<>();
```

Comment:
Here you should specify the lenght of the ArrayList<>() by using new ArrayList<>(selectedRecords.length)

Reply:
you are right!